pkg/upgrade/control_plane.go

```diff
@@ -13,19 +13,17 @@ import (
 
 	"sigs.k8s.io/yaml"
 
-	"k8s.io/apimachinery/pkg/types"
-
 	"github.com/blang/semver"
-	jsonpatch "github.com/evanphx/json-patch"
 	"github.com/go-logr/logr"
 	"github.com/pkg/errors"
 	"github.com/vmware/cluster-api-upgrade-tool/pkg/internal/kubernetes"
 	v1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	clusterapiv1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
-	"sigs.k8s.io/cluster-api/pkg/controller/noderefutil"
+	clusterapiv1alpha2 "sigs.k8s.io/cluster-api/api/v1alpha2"
```

Comment:
we've been naming these `clusterv1` as a convention. That way we don't have to change the references when we change the import path and it's consistent across cluster-api codebases